docs/DeveloperGuide.md

```diff
@@ -1225,22 +1226,143 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### 7.2 Adding a patient
 
-### 7.2 Deleting a patient
+1. Add a patient while all patients are being shown
+
+   1. Prerequisites: List all patients using the `list` command. Multiple patients in the list.
+
+   1. Test case: `add id/ 54321 n/ John Doe p/ John f/ Curry chicken c/ Stable h/ Singing karaoke t/ amnesia`<br>
```

Comment:
Should we also include more test cases such as invalid input value format? Eg. `add id/ABC123 n/ John Doe p/ John f/ Curry chicken c/ Stable h/ Singing karaoke t/ amnesia` since `id/` only accepts numeric characters

Reply:
Since this is just a sample for them to get started, I provided these 2 incorrect add commands to try: `add`, `add id/ 54321 h/ Singing karaoke`<be>
   
That should be ok right

Reply:
yep lgtm!